apps/sps-ordbokene-web/components/Inflection/Table.vue

```diff
@@ -28,23 +26,23 @@
                 <th v-if="hasSing"
                     id="SingInd"
                     class="infl-label sub label-border-bottom" scope="col" :class="mq">
-                  {{tagToName('Ind')}} form
+                  {{tagToName('Ind')}} {{tagToName('Form')}}
```

Comment:
This works for now because English, Norwegian and Ukrainian happen to have the same word order in this instance. Someone expressed interest in translating to Portuguese, which wouldn't work here: "forma indefinida". I think we have similar issues many places in the codebase.

When combining strings from translations we shold pass it as keywords, e. g.:
https://github.com/uib-ub/uib-ub-monorepo/blob/f269272fb36326c1fe9b7e796ef034666a29fa16/apps/sps-ordbokene-web/locales/eng.json#L191 

If we need to add html tags, we can use templates like this:
https://github.com/uib-ub/uib-ub-monorepo/blob/f269272fb36326c1fe9b7e796ef034666a29fa16/apps/sps-ordbokene-web/components/ArticleFooter.vue#L20

If not, we can simply pass it as options:
https://github.com/uib-ub/uib-ub-monorepo/blob/408e2f229dc35a8f78aad0fbaebb9f3f0600ca7b/apps/sps-ordbokene-web/components/AdvancedResults.vue#L34

We can add plural with pipe and the second argument in $t:
https://github.com/uib-ub/uib-ub-monorepo/blob/408e2f229dc35a8f78aad0fbaebb9f3f0600ca7b/apps/sps-ordbokene-web/locales/eng.json#L5
https://github.com/uib-ub/uib-ub-monorepo/blob/408e2f229dc35a8f78aad0fbaebb9f3f0600ca7b/apps/sps-ordbokene-web/components/Autocomplete.vue#L264

Reply:
> This works for now because English, Norwegian and Ukrainian happen to have the same word order in this instance. Someone expressed interest in translating to Portuguese, which wouldn't work here: "forma indefinida". I think we have similar issues many places in the codebase.
> 
> When combining strings from translations we shold pass it as keywords, e. g.:
> 
> https://github.com/uib-ub/uib-ub-monorepo/blob/f269272fb36326c1fe9b7e796ef034666a29fa16/apps/sps-ordbokene-web/locales/eng.json#L191
> 
> If we need to add html tags, we can use templates like this:
> 
> https://github.com/uib-ub/uib-ub-monorepo/blob/f269272fb36326c1fe9b7e796ef034666a29fa16/apps/sps-ordbokene-web/components/ArticleFooter.vue#L20
> 
> If not, we can simply pass it as options:
> 
> https://github.com/uib-ub/uib-ub-monorepo/blob/408e2f229dc35a8f78aad0fbaebb9f3f0600ca7b/apps/sps-ordbokene-web/components/AdvancedResults.vue#L34
> 
> We can add plural with pipe and the second argument in $t:
> 
> https://github.com/uib-ub/uib-ub-monorepo/blob/408e2f229dc35a8f78aad0fbaebb9f3f0600ca7b/apps/sps-ordbokene-web/locales/eng.json#L5
> 
> 
> https://github.com/uib-ub/uib-ub-monorepo/blob/408e2f229dc35a8f78aad0fbaebb9f3f0600ca7b/apps/sps-ordbokene-web/components/Autocomplete.vue#L264

Yes, we have to make it more dynamic.